src/components/RegionMap/RegionMap.tsx

```diff
@@ -18,18 +17,27 @@ const RegionMap: React.FC<{
   const [tooltipContent, setTooltipContent] = useState('');
 
   const countyFipsList = useMemo(() => getCountyFipsList(region), [region]);
+
+  const { result: allCountiesTopoJson } = useCountyGeographies();
+  const { result: statesTopoJson } = useStateGeographies();
+
   const countiesTopoJson = useMemo(
-    () => buildCountyGeometries(countyFipsList),
-    [countyFipsList],
-  );
-  const projectionConfig = useMemo(
-    () => getProjectionConfig(countiesTopoJson, width, height),
-    [countiesTopoJson, width, height],
+    () =>
+      allCountiesTopoJson
+        ? buildCountyGeometries(allCountiesTopoJson, countyFipsList)
+        : null,
+    [countyFipsList, allCountiesTopoJson],
   );
 
-  const statesTopoJson = useMemo(() => getStateGeometries(countyFipsList), [
-    countyFipsList,
-  ]);
+  const projectionConfig = useMemo(() => {
+    return countiesTopoJson
+      ? getProjectionConfig(countiesTopoJson, width, height)
+      : null;
+  }, [countiesTopoJson, width, height]);
+
+  if (!statesTopoJson || !countiesTopoJson || !projectionConfig) {
+    return null;
```

Comment:
What is the effect of returning null here is there just empty content? Potentially could use something like suspense suspense.  When this is null will we want to have filler?

Reply:
Returning `null` doesn't render anything. In this case, I think it's okay because the MiniMap only renders open by default when the screen width is over 1350px, and even in that case, they only see the map as empty while the location page loads.